packages/acp4/acp4.1.0.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+license: "GPL-3.0-only"
+homepage: "https://github.com/tsudalab/ACP4"
+dev-repo: "git+https://github.com/tsudalab/ACP4.git"
+bug-reports: "https://github.com/UnixJunkie/ACP4/issues"
+build: ["dune" "build" "-p" name "-j" jobs]
+install: [
+  ["cp" "bin/acp4_ph4.py" "%{bin}%/acp4_ph4.py"]
+]
+depends: [
+  "batteries"
```

Comment:
```suggestion
  "batteries" {>= "3.3.0"}
```
due to
```
#=== ERROR while compiling acp4.1.0.1 =========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://github.com/tsudalab/ACP4/archive/refs/tags/v1.0.1.tar.gz)
# path                 ~/.opam/4.08/.opam-switch/build/acp4.1.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p acp4 -j 255
# exit-code            1
# env-file             ~/.opam/log/acp4-7-53bf35.env
# output-file          ~/.opam/log/acp4-7-53bf35.out
### output ###
# File "/home/opam/.opam/4.08/lib/bst/bst.dune", line 1, characters 0-0:
# Warning: .dune files are ignored since 2.0. Reinstall the library with dune
# >= 2.0 to get rid of this warning and enable support for the subsystem this
# library provides.
#       ocamlc src/.acp4.eobjs/byte/dune__exe__Common.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.acp4.eobjs/byte -I /home/opam/.opam/4.08/lib/batteries -I /home/opam/.opam/4.08/lib/bst -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/cpm -I /home/opam/.opam/4.08/lib/dolog -I /home/opam/.opam/4.08/lib/equeue -I /home/opam/.opam/4.08/lib/line_oriented -I /home/opam/.opam/4.08/lib/minicli -I /home/opam/.opam/4.08/lib/netcamlbox -I /home/opam/.opam/4.08/lib/netmulticore -I /home/opam/.opam/4.08/lib/netplex -I /home/opam/.opam/4.08/lib/netstring -I /home/opam/.opam/4.08/lib/netsys -I /home/opam/.opam/4.08/lib/num -I /home/opam/.opam/4.08/lib/ocaml/threads -I /home/opam/.opam/4.08/lib/parany -I /home/opam/.opam/4.08/lib/rpc -I /home/opam/.opam/4.08/lib/vector3 -no-alias-deps -open Dune__exe -o src/.acp4.eobjs/byte/dune__exe__Common.cmo -c -impl src/common.ml)
# File "src/common.ml", line 45, characters 23-39:
# 45 |       let nb_actives = A.count_matching fst score_labels in
#                             ^^^^^^^^^^^^^^^^
# Error: Unbound value A.count_matching
#       ocamlc src/.acp4.eobjs/byte/dune__exe__Ph4.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.acp4.eobjs/byte -I /home/opam/.opam/4.08/lib/batteries -I /home/opam/.opam/4.08/lib/bst -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/cpm -I /home/opam/.opam/4.08/lib/dolog -I /home/opam/.opam/4.08/lib/equeue -I /home/opam/.opam/4.08/lib/line_oriented -I /home/opam/.opam/4.08/lib/minicli -I /home/opam/.opam/4.08/lib/netcamlbox -I /home/opam/.opam/4.08/lib/netmulticore -I /home/opam/.opam/4.08/lib/netplex -I /home/opam/.opam/4.08/lib/netstring -I /home/opam/.opam/4.08/lib/netsys -I /home/opam/.opam/4.08/lib/num -I /home/opam/.opam/4.08/lib/ocaml/threads -I /home/opam/.opam/4.08/lib/parany -I /home/opam/.opam/4.08/lib/rpc -I /home/opam/.opam/4.08/lib/vector3 -no-alias-deps -open Dune__exe -o src/.acp4.eobjs/byte/dune__exe__Ph4.cmo -c -impl src/ph4.ml)
# File "src/ph4.ml", line 284, characters 8-20:
# 284 |         L.unfold_exn (fun () ->
#               ^^^^^^^^^^^^
# Error: Unbound value L.unfold_exn
# Hint: Did you mean unfold_exc?
```